jee-commons/src/main/java/biz/paluch/jee/commons/JndiLookupStrategy.java

```diff
@@ -16,8 +16,7 @@
     public <T> T doLookup(String name) {
         try {
             return InitialContext.doLookup(name);
-        } catch (NamingException e) {
-            throw new IllegalArgumentException(e);
-        }
+        } catch (NamingException e) {}
```

Comment:
True. You found a bug in my proposal. You're totally right, the right exception would have been `IllegalArgumentException`.